src/algorithms/pathfinders/minheap.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Implements simple minheap priority queue for the Dijkstra algorithm.
+ */
+export default class MinHeap {
+    data: IEntry[];
+
+    constructor() {
+        this.data = new Array<IEntry>();
```

Comment:
```suggestion
        this.data = [];
```
This is just a shorter and more convenient way how to define empty array.